Exiled.API/Features/Items/Firearm.cs

```diff
@@ -118,9 +118,27 @@ public byte Ammo
         }
 
         /// <summary>
-        /// Gets the max ammo for this firearm.
+        /// Gets or sets the max ammo for this firearm.
         /// </summary>
-        public byte MaxAmmo => Base.AmmoManagerModule.MaxAmmo;
+        public byte MaxAmmo
+        {
+            get => Base.AmmoManagerModule.MaxAmmo;
+            set
+            {
+                switch (Base.AmmoManagerModule)
+                {
+                    case TubularMagazineAmmoManager tubularMagazineAmmoManager:
+                        tubularMagazineAmmoManager.MaxAmmo = value;
+                        break;
+                    case ClipLoadedInternalMagAmmoManager clipLoadedInternalMagAmmoManager:
+                        clipLoadedInternalMagAmmoManager.MaxAmmo = value;
+                        break;
+                    case AutomaticAmmoManager automaticAmmoManager:
```

Comment:
This switch should have a default clause and throw an exception if it's a module that cannot be set.

Reply:
The only default would be with Disruptor than MaxAmmo is actually useless

Reply:
I'm aware, which is why it should throw an error as opposed to not doing anything, as you cannot set the max ammo in this case.